packages/modules/viewer-react/src/components/BaseViewer.tsx

```diff
@@ -29,9 +29,10 @@ export const BaseViewer = ({
   });
 
   const accessToken = useAccessToken();
+  const isBlankConnection = loaderProps.extents && loaderProps.location;
```

Comment:
if its a blank connection and an iTwinId is provided we do want an authclient; the itwinid is used to fetch & display reality data, which is often the most common use case for BlankConnections


suggested reading:
- https://www.itwinjs.org/learning/frontend/blankconnection/
- https://www.itwinjs.org/reference/core-frontend/imodelconnection/blankconnection/
- https://www.itwinjs.org/reference/core-frontend/imodelconnection/blankconnectionprops/